src/content/en/tools/chrome-devtools/settings.markdown

```diff
@@ -0,0 +1,114 @@
+---
+layout: shared/narrow
+title: "Configure and Customize DevTools"
+description: "Change the appearance of DevTools and access hidden features."
+published_on: 2016-03-29
+updated_on: 2016-03-29
+order: 1
+authors:
+  - kaycebasques
+translation_priority: 0
+key-takeaways:
+  tldr:
+    - "Open the Main and Settings menus."
+    - "Customize the appearance of DevTools."
+    - "Access hidden features."
+---
+
+<p class="intro">Change the appearance of DevTools and access hidden 
+features.</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.tldr %}
+
+## Open the Main Menu {#main-menu}
+
+The **Main Menu** of DevTools is a dropdown menu for configuring how
+DevTools looks, accessing additionals tools, opening the Settings, and more.
+
+To open the Main Menu, click on the **Main Menu** button in the top-right
+of the DevTools window.
+
+![main menu](images/main-menu.png)
+
+## Open the Settings {#settings}
+
+To open the DevTools Settings, press <kbd>F1</kbd> while DevTools is in focus,
+or [open the Main Menu](#main-menu) and then select **Settings**.
+
+## Reorder panel tabs {#panel-tabs}
+
+Click, hold, and drag a panel tab to change its ordering. Your custom tab order
+persists across DevTools sessions.
+
+For example, by default the **Network** tab is usually the fourth to the left.
+
+![before reorder](images/before-reorder.png)
+
+You can drag it to any position, such as the first to the left.
+
+![after reorder](images/after-reorder.png)
+
+## Customize DevTools placement {#placement}
+
+You can dock DevTools on the bottom of the page, to right of the page, or 
+you can open it in a new window. 
+
+To change the DevTools' placement, [open the Main Menu](#main-menu) and select
+the **Undock into separate window** 
+(![undock button](images/undock.png){:.inline})
+button, **Dock to bottom** 
+(![dock to bottom button](images/dock-bottom.png){:.inline})
+button, or 
+**Dock to right** 
+(![dock to right button](images/dock-right.png){:.inline})
+button. 
+
+## Set dark theme {#dark-theme}
+
+To use a black and orange DevTools theme, [open DevTools' Settings](#settings),
```

Comment:
Nothing really orange about it. Orange are just a few of the inverted colors. I'd prefer not to mention orange, as it might change.